tests/tests_end_to_end/test_rules_flow/test_to_extension_transformer.yml

```diff
@@ -2561,11 +2576,17 @@ enterprise:
     neatId: http://purl.org/cognite/neat/data-model/verified/physical/cdf_cdm/CogniteCore/v1/CogniteFileCategory
     view: cdf_cdm:CogniteFileCategory(version=v1)
   - description: Navigational aid for traversing CognitePointCloudModel instances
+    filter_: 'rawFilter({"and": [{"hasData": [{"type": "container", "space": "cdf_cdm_3d",
+      "externalId": "Cognite3DModel"}]}, {"equals": {"property": ["cdf_cdm_3d", "Cognite3DModel",
+      "type"], "value": "PointCloud"}}]})'
     implements: cdf_cdm:CogniteDescribable(version=v1),cdf_cdm:Cognite3DModel(version=v1)
     in_model: true
     neatId: http://purl.org/cognite/neat/data-model/verified/physical/cdf_cdm/CogniteCore/v1/CognitePointCloudModel
     view: cdf_cdm:CognitePointCloudModel(version=v1)
   - description: Navigational aid for traversing CognitePointCloudRevision instances
+    filter_: 'rawFilter({"and": [{"hasData": [{"type": "container", "space": "cdf_cdm_3d",
+      "externalId": "Cognite3DRevision"}]}, {"equals": {"property": ["cdf_cdm_3d",
+      "Cognite3DModel", "type"], "value": "PointCloud"}}]})'
```

Comment:
Seems we are not handling the combination of HasData and Equals Node type filter. Maybe something we should do